datasafe-rest-impl/src/main/resources/application.properties

```diff
@@ -3,3 +3,19 @@ management.endpoints.web.exposure.include=*
 datasafe.systemRoot=s3://adorsys-docusafe/datasafe/
 datasafe.keystorePassword=PAZZWORD
 datasafe.bucketName=${AWS_BUCKET}
+
+datasafe.amazonSecretAccessKey=+M+lKaGKyv2aA3Kh1cjVbtoywsS+mQv+B7RWZVq+
```

Comment:
I bet you want to clear these values and change credentials if it was public resource